src/LineClient.AspNetCore.Messaging/Models/LineMessage/Implements/LineTextMessage.cs

```diff
@@ -1,12 +1,33 @@
 ﻿using LineClient.AspNetCore.Messaging.Models.LineMessage.Interfaces;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Net.Http;
+using System.Text;
 
 namespace LineClient.AspNetCore.Messaging.Models.LineMessage.Implements
 {
     public class LineTextMessage : ILineTextMessage
     {
-        public string GenerateStringContent()
+        private readonly string toLineUserId;
+        private readonly string message;
+        public LineMessageText (string toLineUserId, string message)
         {
-            throw new System.NotImplementedException();
+            this.toLineUserId = toLineUserId;
+            this.message = message;
         }
+        public StringContent GenerateStringContent()
+        {
+            dynamic JsonObj = new ExpandoObject();
+            JsonObj.messages = new List<ExpandoObject>();
+            JsonObj.to = toLineUserId;
+            dynamic JsonObj_Message = new ExpandoObject();
+            JsonObj_Message.type = "text";
+            JsonObj_Message.text = message;
+            JsonObj.messages.Add(JsonObj_Message);
+            string JsonObjSerialize = JsonConvert.SerializeObject(JsonObj);
+            StringContent stringContent = new StringContent(JsonObjSerialize, Encoding.UTF8, "application/json");
+            return stringContent;
```

Comment:
```suggestion
            return StringContent stringContent = new StringContent(JsonObjSerialize, Encoding.UTF8, "application/json");
```
ทำแบบนี้แทนดีกว่า Code สะอาด และไม่เปลือง RAM สำหรับ Local Variable